<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerVideos</class>
 <widget class="QMainWindow" name="playerVideos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>playerVideos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;Arial&quot;;
background-color: ghostwhite;
border:1px solid beige;
color:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="format">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="saving">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>6</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>song</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>540</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="progressdisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="duration">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>04:47:55</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>540</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="volume">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="displayvolume">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="imageDisplayer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height:598;
 width:398;
padding:auto;
background-color: rgb(51, 51, 51);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color:floralwhite;
color:black;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="isWrapping" stdset="0">
     <bool>false</bool>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="batchSize">
     <number>20</number>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previous">
       <property name="styleSheet">
        <string notr="true">  opacity: 1;
  width: 50px;
  height: 80px;
  background: #4b00ff;
  position: absolute;
  right: 5px;
  top: -40px;
  border-left: 20px solid #4b00ff;</string>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/ic/iconsFolder/previous.png</normalon>
        </iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play">
       <property name="styleSheet">
        <string notr="true">  opacity: 1;
  width: 50px;
  height: 80px;
  background: #4b00ff;
  position: absolute;
  right: 5px;
  top: -40px;
  border-left: 20px solid #4b00ff;
 </string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/ic/iconsFolder/play.png</normalon>
        </iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paused">
       <property name="styleSheet">
        <string notr="true">  opacity: 1;
  width: 50px;
  height: 80px;
  background: #4b00ff;
  position: absolute;
  right: 5px;
  top: -40px;
  border-left: 20px solid #4b00ff;</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/ic/iconsFolder/pause.png</normalon>
        </iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nexts">
       <property name="styleSheet">
        <string notr="true">  opacity: 1;
  width: 50px;
  height: 80px;
  background: #4b00ff;
  position: absolute;
  right: 5px;
  top: -40px;
  border-left: 20px solid #4b00ff;</string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/ic/iconsFolder/next.png</normalon>
        </iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="repeat">
       <property name="styleSheet">
        <string notr="true">  opacity: 1;
  width: 50px;
  height: 80px;
  background: #4b00ff;
  position: absolute;
  right: 5px;
  top: -40px;
  border-left: 20px solid #4b00ff;</string>
       </property>
       <property name="text">
        <string>Repreat</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/ic/iconsFolder/repeat.png</normalon>
        </iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titleMusic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sizeMusic">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>466</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="typeMusic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>466</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dateMusic">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>466</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MusicSytemowner">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>466</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusic_File">
    <property name="title">
     <string>Music File</string>
    </property>
    <addaction name="actionAdd_File"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_PlayList"/>
    <addaction name="separator"/>
    <addaction name="actionView_PlayList"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_PlayList"/>
    <addaction name="separator"/>
    <addaction name="actionExit_Player"/>
    <addaction name="separator"/>
    <addaction name="actionAuto_PlayList"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionPlay_3"/>
    <addaction name="separator"/>
    <addaction name="actionPause_3"/>
    <addaction name="separator"/>
    <addaction name="actionPrevious_2"/>
    <addaction name="separator"/>
    <addaction name="actionNext_2"/>
    <addaction name="separator"/>
    <addaction name="actionRepeat"/>
    <addaction name="separator"/>
    <addaction name="actionStop_3"/>
   </widget>
   <widget class="QMenu" name="menuThemes">
    <property name="title">
     <string>Themes</string>
    </property>
    <addaction name="actionSelect_theme"/>
    <addaction name="separator"/>
    <addaction name="actionDefault_theme"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOnline">
    <property name="title">
     <string>Online</string>
    </property>
    <addaction name="actionMusic_link"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMusic_File"/>
   <addaction name="menuActions"/>
   <addaction name="menuThemes"/>
   <addaction name="menuOnline"/>
  </widget>
  <action name="actionAdd_File">
   <property name="icon">
    <iconset>
     <normalon>:/ic/iconsFolder/add-song-interface-symbol.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add Music</string>
   </property>
  </action>
  <action name="actionAdd_PlayList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/playlist.png</normaloff>:/ic/iconsFolder/playlist.png</iconset>
   </property>
   <property name="text">
    <string>Add PlayList</string>
   </property>
  </action>
  <action name="actionView_PlayList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/playlist.png</normaloff>:/ic/iconsFolder/playlist.png</iconset>
   </property>
   <property name="text">
    <string>View PlayList</string>
   </property>
  </action>
  <action name="actionRemove_PlayList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/music (2).png</normaloff>:/ic/iconsFolder/music (2).png</iconset>
   </property>
   <property name="text">
    <string>Remove PlayList</string>
   </property>
  </action>
  <action name="actionExit_Player">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/cross.png</normaloff>:/ic/iconsFolder/cross.png</iconset>
   </property>
   <property name="text">
    <string>Exit Player</string>
   </property>
  </action>
  <action name="actionAuto_PlayList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/playlist.png</normaloff>:/ic/iconsFolder/playlist.png</iconset>
   </property>
   <property name="text">
    <string>Auto PlayList</string>
   </property>
  </action>
  <action name="actionPlay_3">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/play.png</normaloff>:/ic/iconsFolder/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause_3">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/pause.png</normaloff>:/ic/iconsFolder/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionPrevious_2">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/previous.png</normaloff>:/ic/iconsFolder/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="actionNext_2">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/next.png</normaloff>:/ic/iconsFolder/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/repeat.png</normaloff>:/ic/iconsFolder/repeat.png</iconset>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="actionStop_3">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/ic/iconsFolder/stop-button.png</normaloff>:/ic/iconsFolder/stop-button.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSelect_theme">
   <property name="text">
    <string>Select theme</string>
   </property>
  </action>
  <action name="actionDefault_theme">
   <property name="text">
    <string>Default theme</string>
   </property>
  </action>
  <action name="actionMusic_link">
   <property name="text">
    <string>Music link</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
